### ISSUE TITLE
Any documents for APIs?

### ISSUE THREAD

**Issue by rookiejune** [lixilinx/psgd_torch#4]
I am trying to find a torch-version for tensorflow_probability.optimizer.StochasticGradientLangevinDynamics and wondering if there are any documents for pSGLD...

**Comment by opooladz** [comment]
I think you have come to the wrong place. This is preconditioned stochastic gradient descent. We are not explicitly doing any mcmc although you can use this repo to do do ur own HMC and RMHMC.

**Comment by lixilinx** [comment]
Added docstrings to classes KronWhiten, KronNewton, LRAWhiten and LRANewton. 

Always good to check [here](https://arxiv.org/abs/2402.11858) for math details.

Thanks!
